src/mercedestrenz/data.py

```diff
@@ -0,0 +1,18 @@
+from importlib import resources
+import pandas as pd
+
+def get_listings():
```

Comment:
Two things:
1. Perhaps make this function more informative of what it's actually doing as in loading the sample data
2. I would add a type hint of what's returned like this:
```suggestion
def load_sample_mercedes_listings()-> pd.DataFrame():
```